docs/fp_primer.md

```diff
@@ -487,30 +524,288 @@ This algorithmically looks something like this:
+
+If instead of `1` and `2`, we had `10` and `f32.max`, so the true result is
+`f32.max + 10`, the user will see a difference. Specifically the runtime
+evaluated version will still run, but the result in `c` will be an indeterminate
+value, which is any finite f32 value. For the compile time example instead,
+compiling the shaderwill fail validation.
+
+This applies to any operation, and isn't restricted to just addition. Anytime a
+value goes outside the finite values the shader will hit these results,
+indeterminate for runtime execution and validation failure for compile time
+execution.
+
+Unfortunately we are dealing with intervals of results and not precise results.
+So this leads to more even conceptual complexity. For runtime evaluation, this
+isn't too bad, because the rule becomes if any part of the interval is
+non-finite then an indeterminate value can be a result, and the interval for an
+indeterminate result `[min, max]`, will include any finite portions of the
+interval.
+
+Compile time evaluation becomes significantly more complex, because difference
+isn't what interval is returned, but does this shader compile or not, which are
+mutually exclusive. This is compounded even further by having to consider out of
+bounds vs overflow behaviour. Thankfully this can be broken down into a case by
+case basis based on where an interval falls.
+
+Assuming `X`, is the well-defined result of an operation, i.e. not an
+indeterminate result because the operation isn't defined for the inputs:
+
+| Region                 |                                                  | Result                         |
+|------------------------|--------------------------------------------------|--------------------------------|
+| `abs(X) <= finite max` | interval falls completely in the finite bounds   | validation succeeds            |
+| `abs(X) >= 2^(emax+1)` | interval falls completely in the overflow bounds | validation fails               |
+| Otherwise              | interval intersects the out of bounds region     | validation may succeed or fail |
+
+The final case is somewhat difficult from a CTS perspective, because now it
+isn't sufficient to know that a non-finite result has occurred, but what the
+specific result is needs to be tracked. Additionally, the expected result is
+somewhat ambiguous, since a shader may or may not compile. This could in theory
+still be tested by the CTS, via having switching logic that determines in this
+region, if the shader compiles expect these results, otherwise pass the test.
+This adds a significant amount of complexity to the testing code for thoroughly
+testing a relatively small segment of values. Other environments do not have the
+behaviour in this region as rigorously defined nor tested, so fullly testing
+here would likely find lots of issues that would just need to be mitigated in
+the CTS.
+
+At this point testing this region is not considered high priority and has been
```

Comment:
Avoiding passive voice:

> Currently, we choose to avoid testing validation of near-overflow scenarios.

Reply:
Done

(My background training in physics strikes again. Physics technical writing prefers passive voice, CS prefers active, this caused so many issues when I wrote my thesis.)

Reply:
Yeah, I know.
I write the royal "we" a lot.

In this particular case I wanted it to pop out more that there was an active choice going on here.  To highlight that a different choice is possible, with extra work.